persist/fs/read.go

```diff
@@ -260,7 +271,7 @@ func (r *reader) Read() (ts.ID, []byte, error) {
 
 	r.entriesRead++
 
-	return ts.BinaryID(entry.Id), data, nil
+	return ts.StringID(string(entry.Id)), data, nil
```

Comment:
I think that would be more appropriate here yeah, since right now it looks like an unnecessary string conversion and somebody might accidentally remove it in the future.